docs/manual/wayland.rst

```diff
@@ -51,7 +53,80 @@ Input Device Configuration
+        subprocess.run(home + "/.config/qtile/autostart.sh")
+
+GTK settings aren't applied
+===========================
+
+GTK have a known issue of not recognizing setted variables. You can fix this by saving below code into a script in your script directory. Then you can just put the name of the script into your autostarting script.
+
+Edit: GTK has a known issue of not recognizing set variables.  To fix this, create a script with the code below and save it in your script directory.  Then add the script to your autostart script.  For a GUI solution, the program `nwg-look` works well.
+
+.. note::
+    Please do not forget to place your directory into a PATH and make the script executable
+
+.. code-block:: sh
+    #!/bin/sh
+
+    # usage: import-gsettings
+    config="${XDG_CONFIG_HOME:-$HOME/.config}/gtk-3.0/settings.ini"
+    if [ ! -f "$config" ]; then exit 1; fi
+
+    gnome_schema="org.gnome.desktop.interface"
+    gtk_theme="$(grep 'gtk-theme-name' "$config" | sed 's/.*\s*=\s*//')"
+    icon_theme="$(grep 'gtk-icon-theme-name' "$config" | sed 's/.*\s*=\s*//')"
+    cursor_theme="$(grep 'gtk-cursor-theme-name' "$config" | sed 's/.*\s*=\s*//')"
+    font_name="$(grep 'gtk-font-name' "$config" | sed 's/.*\s*=\s*//')"
+    gsettings set "$gnome_schema" gtk-theme "$gtk_theme"
+    gsettings set "$gnome_schema" icon-theme "$icon_theme"
+    gsettings set "$gnome_schema" cursor-theme "$cursor_theme"
+    gsettings set "$gnome_schema" font-name "$font_name"
+
+Gamma adjustment tool
+=====================
+
+Redshift doesn't work on Wayland, but there are alternatives like: `gammastep <https://gitlab.com/chinstrap/gammastep>`, `wlsunset <https://git.sr.ht/~kennylevinsen/wlsunset>`, `wl-gammarelay <https://github.com/jeremija/wl-gammarelay>` and `wl-gammarelay-rs <https://github.com/MaxVerevkin/wl-gammarelay-rs>`.
+
+Screenshots
+===========
+
+As `scrot <https://github.com/dreamer/scrot>` and other similiar X11 tools don't work on wayland, you might be interested in: `grim <https://git.sr.ht/~emersion/grim>`/`slurp <https://github.com/emersion/slurp>`, `shotman <https://git.sr.ht/~whynothugo/shotman>`, `flameshot <https://flameshot.org/>` or `swappy <https://github.com/jtheoof/swappy>`.
+
+Screen sharing
+==============
+
+Here is a write up on how to get screen sharing enabled in Wayland:
+
+To get screen sharing working on Qtile Wayland, make sure `xdg-desktop-portal`, `xdg-desktop-portal-gtk`, `xdg-desktop-portal-wlr`, `python-dbus-next`, and `wlroots` are installed on your system and add the following lines to your `config.py`:
+
+.. code-block:: python
+    import os  
+    from libqtile import qtile   
+    if qtile.core.name == "wayland":  
+        os.environ["XDG_SESSION_DESKTOP"] = "qtile:wlroots"
+        os.environ["XDG_CURRENT_DESKTOP"] = "qtile:wlroots"
+
+And add this to your autostart script:
+
+.. code-block:: sh
+    dbus-update-activation-environment --systemd WAYLAND_DISPLAY XDG_CURRENT_DESKTOP &
```

Comment:
this is not guaranteed to work as you would also have to restart your systemd services to take effect, see step 5 here https://github.com/emersion/xdg-desktop-portal-wlr/wiki/%22It-doesn't-work%22-Troubleshooting-Checklist